packages/ui-components/src/components/relative-time-picker/relative-time-picker.scss

```diff
@@ -0,0 +1,41 @@
+@import '../../assets/styles/globals';
+
+:host {
+	display: block;
+}
+
+.relative-time-container {
+	@include kelvin-night-scrollbar();
+	display: flex;
+	flex-direction: column;
+	width: 328px;
+	max-height: 330px;
+	background-color: kv-color('neutral-7');
+
+	.relative-time-selector {
+		padding: $spacing-4x 0;
+		display: flex;
+		flex-direction: column;
+		gap: $spacing-3x;
+		overflow: auto;
+
+		&--has-scroll {
+			kv-time-picker-select-option {
+				--select-option-padding: #{0 $spacing 0 $spacing-4x};
+			}
+		}
+	}
+
+	.selectable {
+		border-top: 1px solid kv-color('neutral-6');
+		padding: $spacing 0;
+
+		.mock {
```

Comment:
is this necessary?

Reply:
Have to check! Probably not and forgot to remove 👍